### ISSUE TITLE
Can't make folder when trying to save a preset

### ISSUE THREAD

**Issue by grammoboy2** [Geonkick-Synthesizer/geonkick#139]
There is no option to make a folder when saving the preset

**Comment by iurienistor** [comment]
Yes, that would be a good feature. I'll try to implement this in the next releases.

**Comment by iurienistor** [comment]
@grammoboy2 If you are interested in this, and want to develop this feature, I can assign you this issue. :)

If anyone interested in developing this feature, please, leave here a comment.
